data/human/human missions.txt

```diff
@@ -3996,3 +3996,285 @@ mission "Fumbling Mugger"
 event "fumbling mugger followup timer"
 
 
+
+mission "Dangerous Games 2"
+	name "Karengo's Galactic Adventures"
+	description "The adventurer Karengo and five others are traveling with you as part of his "Galactic Adventures". Bring Karengo and crew to <destination>."
+	minor
+	passengers 9
+	to offer
+		random < 15
+		has "Dangerous Games 1: done"
+	source 
+		attributes "frontier" "pirate" "rim"
+	cargo "subpods" 9
+	destination Poseidos
```

Comment:
```suggestion
	source 
		attributes "frontier" "pirate" "rim"
	destination Poseidos
	cargo "subpods" 9
	passengers 9
	to offer
		random < 15
		has "Dangerous Games 1: done"
```
iirc this is the correct order according to TMBR missions. Or at very least `to offer` should be last in here and source-destination and cargo-passenger lines should be next to each other.